yellowbrick/text/tsne.py

```diff
@@ -177,6 +177,8 @@ def __init__(self, ax=None, decompose='svd', decompose_by=50, labels=None,
         self.colors = colors
         self.colormap = colormap
         self.random_state = random_state
+        self._size = kwargs.pop('size', None)
```

Comment:
Note that there is a size `@property` that is part of the base class that uses this attribute; unfortunately, it isn't as simple as simply setting `_size` to what the user passes in, a computation has to be made: 

https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/base.py#L100

I made sort of a strange choice when advising the `size` property implementation - based on my matplotlib naivete and my familiarity with pixel sizes, I requested that size be specified in pixels instead inches as is usual in matplotlib. This has had some implications for our implementation and we should probably discuss whether or not we'd like to revise it.